doc/developer/platform/ux.md

```diff
@@ -1087,7 +1087,7 @@ was Snowflake's approach.
 [dbt]: https://www.getdbt.com
 [frontegg-api]: https://docs.frontegg.com/reference/getting-started-with-your-api
 [frontegg]: https://frontegg.com
-[global-api-spec]: https://cloud.materialize.com/api/schema/browse
+[global-api-spec]: https://console.materialize.com/api/schema/browse
```

Comment:
I'm not sure where we currently host this (maybe @evanharmon does), but it is not hosted under `console.*`.

Reply:
The updated change to `<https://api.cloud.materialize.com>` is correct.

Reply:
But there is now schema/browse there: https://api.cloud.materialize.com/schema/browse

Reply:
I can confirm the schema isn't publicly available for the global-api.

Reply:
So we should just remove this link entirely from the ux page?

Reply:
I think that's for the best.